app/assets/stylesheets/application.scss

```diff
@@ -24,8 +24,17 @@ main {
   @include inner-block;
 }
 
+.print-page {
+  display: none;
+  margin-top:$gutter;
+
+  .js-enabled & {
+    display: block;
+  }
+}
```

Comment:
There was a reason at first, before I used the .js-enabled & method, but that seems to work fine now. So I've updated this.